### ISSUE TITLE
PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25

### ISSUE THREAD

**Issue by ellie** [PostHog/charts-clickhouse#531]
```
PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25
```

## Bug description

This warning is shown while applying our current helm chart to a v1.21+ k8s cluster. We should ensure our dependency charts are upgraded before v1.25

## Expected behavior

No warnings :)

## Actual behavior

There is a warning

## How to reproduce

1. Apply the chat to a k8s cluster running v1.21+

## Environment

- [ ] Deployment platform (AWS/Azure/DigitalOcean/GCP/...): _please provide_
- [ ] k8s version (server version from `kubectl version`): _please provide_
- [ ] Chart version/commit (`helm list -n posthog`): _please provide_
- [ ] Posthog version: _please provide_

**Comment by guidoiaquinti** [comment]
To add some context: those are coming from upstream chart dependencies (Grafana repo). See upstream issue https://github.com/grafana/helm-charts/issues/748

**Comment by guidoiaquinti** [comment]
This is now fixed ✅  v1.25 works well in this chart despite [its support has not landed yet](https://github.com/PostHog/charts-clickhouse/pull/569)
